### PR TITLE
Calculator

### PR DESCRIPTION
**Pull request by Sajeelzafar**
### In this pull request, the following tasks have been completed:

- Created a component directory and made added a calculator.js file.
- Build a layout of calculator and completed basic styling.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,29 +1,9 @@
...
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+  return buttons();
 }
```
Comment (ShahierNashaat) [inline src/App.js]: - [x] Kindly notice that the `App.js` component should be class-based.
```javascript
// src/App.js

import React, { Component } from 'react';
import Calculator from ./components; // Import Calculator component

class App extends Component {
  render() {
    // reference as a js object
    return {
      <Calculator />
    };
  }
}
```

File: src/components/calculator.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        <li className="bigButton flex">0</li>
+        <li className="simpleButtons flex">.</li>
+        <li className="colorButtons flex">=</li>
+      </ul>
+
+    </div>
+  </section>
+);
+
+export default buttons;
```
Comment (ShahierNashaat) [inline src/components/calculator.js]: - [x] Kindly notice that the `Calculator.js` component should be class-based, also you should name the component `Calculator`.
```javascript
// ./src/components/Calculator.js

import React, { Component } from 'react';

class Calculator extends Component {
  render() {
    // reference as a js object
    return {
      <div>
       // Calculator app design goes here
      </div>
    };
  }
}

export default Calculator; // export Calculator component
```

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @Sajeelzafar, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 

- Correct use of GitFlow ✔️
- Descriptive commit messages ✔️
- Professional looking pull request ✔️
- Your readme looks professional ✔️
- Good overall design ✔️
- Your code is clean and well documented ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Sajeelzafar** [comment]
Thank you for your time and comments @ShahierNashaat , I've used your guidelines to convert it into class components and also changed the name of the class, as suggested.

**Review by topeogunleye — APPROVED** [review]
### Hi @Sajeelzafar, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and description :heavy_check_mark:
 
## [Optional] suggestions:
*  Nothing to mention.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**